### PR TITLE
Insert a comment line

### PR DESCRIPTION
**Pull request by khadkarajesh**

### CODE REVIEW

File: app/src/main/java/com/ekbana/weather/weather/Activity/AddHandle.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
 import java.util.ArrayList;
 
 public class AddHandle extends ActionBarActivity {
-
+//manas shrestha's comment
```
Comment (manas-raj-shrestha) [inline app/src/main/java/com/ekbana/weather/weather/Activity/AddHandle.java:23]: very good comment

File: app/src/main/java/com/ekbana/weather/weather/Activity/AddHandle.java [hunk 2]
```diff
@@ -20,7 +20,7 @@
...
     Toolbar toolbar;
```
Comment (khadkarajesh) [inline app/src/main/java/com/ekbana/weather/weather/Activity/AddHandle.java:24]:
```
public class AddHandle extends ActionBarActivity {
public class AddHandle extends ActionBarActivity {
```

fuhghdjga

**Comment by khadkarajesh** [comment]
:ok:

**Comment by khadkarajesh** [comment]
:+1:

**Pull request closed without merging**